nck/utils/formatter.py

```diff
@@ -15,17 +15,17 @@
 # You should have received a copy of the GNU Lesser General Public License
 # along with this program; if not, write to the Free Software Foundation,
 # Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
-from nck.readers import reader_classes
-from nck.writers import writer_classes
+from nck.readers import reader_classes, Reader
+from nck.writers import writer_classes, Writer
 
 
-def format_reader(reader):
+def format_reader(reader: dict) -> Reader:
```

Comment:
For the type hinting, this is not very natural but you need to do this instead:

```suggestion
from typing import Dict

def format_reader(reader: Dict) -> Reader:
```

Thus, we will be able to use tools like mypi to do type checking.

Reply:
Thanks, i didn't know that